AjobK/Athena
# 220 profile banner attachment

closes #220 

 **What is it supposed to do**
Allows users to update their banners by cropping a given image to the required format.

 **How can we test your branch**
Navigate to your profile while logged in. Click on the banner (a floating action button should pop up as well). Try out some images. Maybe try out some images with odd sizes (such as very wide or tall images) as well if you have those. Try to crop it and save it. It should be updated immediately, and this banner should be shown again when you refresh te page.

Merging develop with this branch caused a lot of merge conflicts (hence the rather long delay), so it may be possible I destroyed some other functionalities that I did not notice. The functionalities that caused trouble in the merge conflicts are updating profile avatars and following users. So maybe try to see if it still works (works for me though).

 **What broweser have been checked**
